server/src/main/java/org/apache/gravitino/server/web/rest/PolicyOperations.java

```diff
@@ -0,0 +1,273 @@
+  @Timed(name = "create-policy." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "create-policy", absolute = true)
+  public Response createPolicy(
+      @PathParam("metalake") String metalake, PolicyCreateRequest request) {
+    LOG.info("Received create policy request under metalake: {}", metalake);
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            request.validate();
+            Policy policy =
+                Policy.BuiltInType.fromPolicyType(request.getPolicyType())
+                        == Policy.BuiltInType.CUSTOM
+                    ? policyDispatcher.createPolicy(
+                        metalake,
+                        request.getName(),
+                        request.getPolicyType(),
+                        request.getComment(),
+                        request.getEnabled(),
+                        request.getExclusive(),
+                        request.getInheritable(),
+                        request.getSupportedObjectTypes(),
+                        fromDTO(request.getPolicyContent()))
+                    : policyDispatcher.createPolicy(
+                        metalake,
+                        request.getName(),
+                        request.getPolicyType(),
+                        request.getComment(),
+                        request.getEnabled(),
+                        fromDTO(request.getPolicyContent()));
+
+            LOG.info("Created policy: {} under metalake: {}", policy.name(), metalake);
+            return Utils.ok(new PolicyResponse(DTOConverters.toDTO(policy, Optional.empty())));
+          });
+    } catch (Exception e) {
+      return ExceptionHandlers.handlePolicyException(
+          OperationType.CREATE, request.getName(), metalake, e);
+    }
+  }
+
+  @GET
+  @Path("{policy}")
+  @Produces("application/vnd.gravitino.v1+json")
+  @Timed(name = "get-policy." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "get-policy", absolute = true)
+  public Response getPolicy(
+      @PathParam("metalake") String metalake, @PathParam("policy") String name) {
+    LOG.info("Received get policy request for policy: {} under metalake: {}", name, metalake);
+
+    try {
+      return Utils.doAs(
+          httpRequest,
+          () -> {
+            Policy policy = policyDispatcher.getPolicy(metalake, name);
+            LOG.info("Get policy: {} under metalake: {}", name, metalake);
+            return Utils.ok(new PolicyResponse(DTOConverters.toDTO(policy, Optional.empty())));
+          });
+    } catch (Exception e) {
+      return ExceptionHandlers.handlePolicyException(OperationType.GET, name, metalake, e);
+    }
+  }
+
+  @PUT
+  @Path("{policy}")
+  @Produces("application/vnd.gravitino.v1+json")
+  @Timed(name = "alter-policy." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
+  @ResponseMetered(name = "alter-policy", absolute = true)
+  public Response alterPolicy(
```

Comment:
There are two methods name `alterPolicy`, I believe it's more suitble to differentate them.